src/firefly/js/visualize/saga/CatalogWatcher.js

```diff
@@ -114,7 +114,7 @@ function handleCatalogUpdate(tbl_id) {
     const params= {
         startIdx : 0,
         pageSize : MAX_ROW,
-        inclCols : `${columns.lonCol},${columns.latCol},ROW_IDX`
+        inclCols : `"${columns.lonCol}","${columns.latCol}",ROW_IDX`        // column names should be in quotes
```

Comment:
Only the unknown columns should be in quotes. The columns that we know (like ROW_IDX) are safe, right?

From tis point of view we did not have to add quotes to columns like Peak, Period, etc.

Reply:
Actually, all column names should be in quotes.  But because ROW_IDX is all-caps single word, it does not need to be in quotes.  But, I should quotes them as well for consistency.